amundsen_application/static/js/components/ColumnList/ColumnStats/index.tsx

```diff
@@ -4,78 +4,85 @@
+      <div className="stat-value">{stat_val}</div>
+    </div>
+  );
+};
 
-  renderColumnStat = (entry: TableColumnStats) => {
-    return (
-      <div className="column-stat-row" key={entry.stat_type}>
-        <div className="stat-name body-3">{entry.stat_type.toUpperCase()}</div>
-        <div className="stat-value">{entry.stat_val}</div>
-      </div>
-    );
-  };
+const getStart = ({ start_epoch }) => start_epoch;
+const getEnd = ({ end_epoch }) => end_epoch;
 
-  render = () => {
-    const { stats } = this.props;
-    if (stats.length === 0) {
-      return null;
-    }
+const ColumnStats: React.FC<ColumnStatsProps> = ({
+  stats,
+}: ColumnStatsProps) => {
+  if (stats.length === 0) {
+    return null;
+  }
+  const startEpoch = Math.min(...stats.map(getStart));
+  const endEpoch = Math.max(...stats.map(getEnd));
 
-    // TODO - Move map statements to separate functions for better testing
-    const startEpoch = Math.min(...stats.map((s) => s.start_epoch));
-    const endEpoch = Math.max(...stats.map((s) => s.end_epoch));
+  return (
+    <article className="column-stats">
+      <div className="stat-collection-info">
+        <span className="stat-title">{COLUMN_STATS_TITLE} </span>
+        {getStatsInfoText(startEpoch, endEpoch)}
+      </div>
+      <div className="column-stats-table">
+        <div className="column-stats-column">
+          {stats.map((stat, index) => {
+            if (index % 2 === 0) {
+              return (
+                <ColumnStatRow
+                  key={stat.stat_type}
+                  stat_type={stat.stat_type}
+                  stat_val={stat.stat_val}
+                />
+              );
+            }
 
-    return (
-      <section className="column-stats">
-        <div className="stat-collection-info">
-          <span className="title-3">Column Statistics&nbsp;</span>
-          {this.getStatsInfoText(startEpoch, endEpoch)}
+            return null;
+          })}
         </div>
-        <div className="column-stats-table">
-          <div className="column-stats-column">
-            {stats.map((stat, index) => {
-              if (index % 2 === 0) {
-                return this.renderColumnStat(stat);
-              }
-            })}
-          </div>
-          <div className="column-stats-column">
-            {this.props.stats.map((stat, index) => {
-              if (index % 2 === 1) {
-                return this.renderColumnStat(stat);
-              }
-            })}
-          </div>
+        <div className="column-stats-column">
+          {stats.map((stat, index) => {
+            if (index % 2 === 1) {
+              return (
+                <ColumnStatRow
+                  key={stat.stat_type}
+                  stat_type={stat.stat_type}
+                  stat_val={stat.stat_val}
+                />
+              );
+            }
+
+            return null;
+          })}
         </div>
-      </section>
-    );
-  };
-}
+      </div>
+    </article>
```

Comment:
LGTM! Wanted to know why we changed <section> tag to <article> ?

Reply:
Sure!
`<section/>` elements are basically useless, they don't add anything, while article sections point out parts of the UI that could be showed in other contexts, so basically reusable.